web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js

```diff
@@ -24,18 +24,31 @@ openerp.web_ir_actions_act_window_message = function(instance)
     instance.web.ActionManager.include({
```

Comment:
Hi @lisogallo - Thanks for the submission!

This JavaScript is using the deprecated 8.0 API. While it does work now due to some compatibility imports, this will not be the case going forward. 

There is a new Requires system that must be used in order to upgrade to the 9.0 API. Please take a look at the [Javascript module system overview](https://www.odoo.com/documentation/9.0/reference/javascript.html#web-client) for a full description.

For an high level abstract example, you can also take a look at this [PR upgrading the template module](https://github.com/OCA/maintainer-tools/pull/208/files).